tests/plotting/test_plotting.py

```diff
@@ -71,6 +71,7 @@
 # TODO: once we have a stable release for VTK, remove these.
 HIGH_VARIANCE_TESTS = {
     'test_pbr',
+    'test_set_environment_texture',
```

Comment:
Is this missing the last word?
```suggestion
    'test_set_environment_texture_cubemap',
```